packages/v4/src/content/design-guidelines/demos/card-view/card-view.md

```diff
@@ -0,0 +1,76 @@
+---
+id: Card view
+section: demos
+---
+
+
+A **card view** is a grid of cards in a gallery to facilitate browsing. Card views are typically used to present data set summaries, allowing users to drill down into any card to see more detailed content.
+
+<img src="./img/card-view.png" alt="card view" />
+
+Use a card view to:
+- Display items in a data set that are best identified by a graphic or other visual representation.
+- Make information easier to visualize and compare.
+- Summarize a large number of objects at once within the same page.
+
+Do NOT use a card view to make a large amount of text-based content easier to scan. Use a table or list view instead.
+
+### Background colors for card views
+Card views can be placed on a white or a gray background, depending on your use case.
+
+**Use a white backgroun**d if your content view can change, such as when:
+- A card that opens an expandable panel on the right.
+- A card view that toggles into a table or list view.
+
+<img src="./img/white-background.png" alt="card view with white background" />
+
+**Use a gray background** if the sole purpose of your content view is to display cards.
+
+<img src="./img/gray-background.png" alt="card view with gray background" />
+
+### Card views on mobile
+Cards will size down appropriately based on the built-in CSS responsiveness behaviors. Most of the time, they’ll follow the grid layout, expanding and compressing at certain breakpoints. At the smallest mobile size, they take up almost the entire width of the screen.
+
+<img src="./img/mobile-cards.png" alt="mobile card view" width="375"/>
+
+### Card view actions
+You can customize card actions in a few ways. Depending on your use case, consider customizing inline actions, multiple inline actions, global actions, or action cards within your card view.
+
+#### Inline actions
+If an action can be performed on one card only, place the action in the card’s body or footer. You don’t need to include a checkbox because the actions are card-specific.
+
+- If you have a few cards with single actions, use the [secondary button style](/components/button#variations).
+- If you have multiple cards with single actions, use a [link button](/components/button#variations).
+
+<img src="./img/few-multiple-actions.png" alt="few cards with multiple actions" />  
+
+Inline action cards are commonly used as navigation or drill-down indicators. In this case, present the user action on the card to differentiate this type of navigational card from a standard card. This way, users can easily distinguish the card types if your application includes both.   
```

Comment:
I would maybe add "In this case, present the user action on the card **as opposed to in the toolbar**" (assuming that's what this means) just to clarify a little bit. Especially since at the end it says "if your application includes both" but it's not clear what "both" are? Inline action cards and what?

Reply:
I couldn't understand what this was trying to say, so I just removed the entire paragraph. Seemed more confusing than useful.